### PR TITLE
Adding required jsDelivr files

### PR DESCRIPTION
**Pull request by jevy**

### CODE REVIEW

File: files/youtube-google-analytics/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "youtube-google-analytics",
+  "repo": "lunametrics/youtube-google-analytics",
+  "files": {
+    "include": ["lunametrics-youtube.gtm.js", "lunametrics-youtube.gtm.min.js"]
```
Comment (jimaek) [inline files/youtube-google-analytics/update.json]: The min file is enough, no need for the unminified file.

**Comment by jevy** [comment]
Only the minified version now.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Comment by jevy** [comment]
Squashed!

**Pull request closed without merging**